snorkel/analysis/error_analysis.py

```diff
@@ -30,13 +30,16 @@ def get_label_buckets(*y: np.ndarray) -> Dict[Tuple[int, ...], np.ndarray]:
     >>> buckets = get_label_buckets(Y_gold, Y_pred)
 
     The returned ``buckets[(i, j)]`` is a NumPy array of data point indices with
-    predicted label i and true label j.
+    true label i and predicted label j.
+
+    More generally, the returned indexes (i,j,k,...) match the order of labels passed
```

Comment:
Confusing to use i and j here with a different meaning than i and j just two lines above. How about "The returned indices within each bucket refer to the order of the labels that were passed in."

Reply:
Agreed, this is clearer!